src/storybook/decorators.tsx

```diff
@@ -21,3 +22,14 @@ export const Apollo = (DecoratedStory: () => StoryFnReactReturnType): JSX.Elemen
     <Frame style={{ display: 'inline-block' }}>{DecoratedStory()}</Frame>
   </ApolloProvider>
 )
+
+export const Form = (DecoratedStory: () => StoryFnReactReturnType): JSX.Element => {
```

Comment:
Would be cool to allow to pass vars to `useForm` here

Reply:
good idea. would you prefer it as a curried function or a 2 param function? @anxolin @Velenir 

sth like:
```js
import { Form } from 'decorators'

const FormWithArgs = Form(args)

{
  ...
  decorators: [FormWithArgs],
}
```

Reply:
last commit includes this:

```tsx
export const Form = (args?: UseFormOptions) =>
  function InnerForm(DecoratedStory: () => StoryFnReactReturnType): JSX.Element {
    const methods = useForm(args)
    return (
      <Frame>
        <FormProvider {...methods}>
          <form>{DecoratedStory()}</form>
        </FormProvider>
      </Frame>
    )
  }

export const FormEmpty = Form()
```